spec/claCheckSpec.js

```diff
@@ -0,0 +1,212 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Spec for CLA Check Github Action.
+ */
+
+const claCheckGithubActionModule =
+ require('../actions/src/cla_check/claCheck.js');
+const core = require('@actions/core');
+const dispatcher = require('../actions/src/dispatcher');
+const github = require('@actions/github');
+const { google } = require('googleapis');
+const { OAuth2Client } = require('google-auth-library');
+const pullRequestPayload = require('../fixtures/pullRequestPayload.json');
+const sheetsToken = JSON.stringify(require('../fixtures/token.json'));
+const sheetsCredentials =
+  JSON.stringify(require('../fixtures/credentials.json'));
+
+
+describe('CLA check github action Module', () => {
+  beforeEach(async () => {
+    process.env.SHEETS_CRED = sheetsCredentials;
+    process.env.SHEETS_TOKEN = sheetsToken;
+    process.env.GITHUB_TOKEN = 'sample-token';
+    github.context.eventName = 'pull_request';
+    github.context.payload = pullRequestPayload.payload;
+
+    octokit = {
+      issues: {
+        createComment: jasmine.createSpy('createComment').and.resolveTo({})
+      },
+    };
+
+    Object.setPrototypeOf(github.GitHub, function () {
+      return octokit;
+    });
+
+    spyOnProperty(github.context, 'repo').and.returnValue({
+      owner: pullRequestPayload.payload.repository.owner.login,
+      repo: pullRequestPayload.payload.repository.name,
+    });
+    spyOn(claCheckGithubActionModule, 'claCheckGithubAction').and.callThrough();
+  });
+
+  describe('called for only PR opened or reopened action', () => {
+    beforeEach(() => {
+      spyOn(google, 'sheets').and.returnValue({
+        spreadsheets: {
+          values: {
+            get: jasmine.createSpy('get').and.callFake(async (obj, cb) => {
+              await cb(null, {
+                data: {
+                  values: [['testuser7777']]
+                },
+              });
+            }),
+          },
+        },
+      });
+    });
+
+    it('should be called for PR opened action', async () => {
+      await dispatcher.dispatch('pull_request', 'opened');
```

Comment:
This should and others below should be updated to `dispatcher.dispatch('pull_request_target', 'opened')`

Reply:
DONE